app/src/org/commcare/activities/DispatchActivity.java

```diff
@@ -105,12 +112,22 @@ protected void onSaveInstanceState(Bundle outState) {
         outState.putBoolean(EXTRA_CONSUMED_KEY, shortcutExtraWasConsumed);
     }
 
+    private void checkForChangedCCZ() {
+        Intent i = new Intent(getApplicationContext(), UpdateActivity.class);
+        startActivity(i);
```

Comment:
When this update activity is launched, can the user back out of it? Is that okay?